client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -104,7 +104,7 @@ const BrewItem = createClass({
 			</div>
 			<hr />
 			<div className='info'>
-				{brew.tags ? <>
+				{([].concat(brew.tags)[0]!='') ? <>
```

Comment:
@G-Ambatte Can you remind me what this is doing?

Reply:
As `brew.tags` has previously existed as an empty string, when it is converted to an array, `tags[0]` is `''`. When tags are actually set, `tags[0]` contains actual content. So we can't just check for the existence of tags with `tags.length > 0`, we need to check that `tags[0]` is not equal to an empty string - but still taking into account that `tags` might still be a string, having not yet been converted to an array yet.

Reply:
Ok. Can you add this fix to a new PR? I'm going to close this one because I think we are going to remove Systems anyway and merge into Tags, but this line seems like an important fix on its own.